src/effects/audiounits/AudioUnitEffect.cpp

```diff
@@ -479,9 +535,12 @@ size_t AudioUnitEffect::GetTailSize() const
 }
 #endif
 
-bool AudioUnitEffect::ProcessInitialize(
-   EffectSettings &, sampleCount, ChannelNames chanMap)
+bool AudioUnitInstance::ProcessInitialize(
+   EffectSettings &settings, sampleCount, ChannelNames chanMap)
```

Comment:
This is more about the framework than about this PR, anyway: I think the passed EffectSettings reference should be const? or could in principle the settings be changed by the initialization?

Reply:
Ladspa and Lv2 have the notion of an output control port that the instance may change during processing.  So parts of EffectSettings may in fact be modifiable, and the changes settings in general need to be copied back to main thread.

Remember that: it will be relevant to other bug fixes soon.